lib/types.ts

```diff
@@ -634,11 +635,17 @@ export type PageExtensionSDK = BaseExtensionSDK & {
   ids: Omit<IdsAPI, 'field' | 'entry' | 'contentType'>
 }
 
+export interface AppState {
+  EditorInterface: Record<ContentType['sys']['id'], Omit<EditorInterface, 'sys'>>
```

Comment:
`Omit<EditorInterface, 'sys'>` is not a valid type (e.g. `sidebar: { position: 1}`